internal/attestation/crafter/materials/csafvex_test.go

```diff
@@ -102,7 +104,8 @@ func TestCSAFVEXCraft(t *testing.T) {
 			// Mock uploader
 			uploader := mUploader.NewUploader(t)
 			if tc.wantErr == "" {
-				uploader.On("UploadFile", context.TODO(), tc.filePath).
+				uploader.On("Upload", context.TODO(), mock.Anything,
+					"sha256:c27087147fa040909e0ef1b522386608af545b0a163c30c9f11c3d753676fa44", filepath.Base(tc.filePath)).
```

Comment:
why of this specific digest? which one does it correspond?

Reply:
update: I understand now, this is because now we are using `upload` not `upload file` 

For this, since you do not care about the case when the upload fails, you can just add mock.anything in the digest arg.